src/structures/WidgetMember.js

```diff
@@ -21,9 +21,9 @@ class WidgetMember extends Base {
 
     /**
      * The id of the user. It's an arbitrary number.
-     * @type {string}
+     * @type {number}
      */
-    this.id = data.id;
+    this.id = parseInt(data.id);
```

Comment:
You can keep this as a string! Sorry, was mostly a question about it 👍

Reply:
Oh my bad ! Don't worry x) I'll revert it

Reply:
Drop the parseInt, update JSdoc too to mark it as a string